src/spacy/fortex/spacy/spacy_processors.py

```diff
@@ -95,6 +95,24 @@ def validate_spacy_configs(configs: Config):
                 "'sentence' is necessary in configs.processors for 'tokenize'."
             )
 
+    if "umls_link" in configs.processors:
+        if not (configs.medical_onto_type and configs.umls_onto_type):
+            raise ProcessorConfigError(
+                "Please specify medical and umls link ontology types!"
+            )
+
+        entry_type = get_class(configs.medical_onto_type)
+        if not isinstance(entry_type, Annotation) and not issubclass(
+            entry_type, Annotation
+        ):
+            raise ValueError("medical_onto_type must be an Annotation type.")
```

Comment:
to write better exceptions, we should print out which string is not an Annotation type.